README.md

```diff
@@ -18,7 +18,7 @@ use League\Fractal\Manager;
 use League\Fractal\Resource\Collection;
 
 $books = [
-   ['id'=>1, 'title'=>'Hogfather', 'characters' => [...]], 
+   ['id'=>1, 'title'=>'Hogfather', 'characters' => [...]],
```

Comment:
I didn't mean to commit that. It's my editor that trims trailing whitespace. I guess there is nothing wrong with this being stripped in this PR, but if you want, I can make a new "Clean-up" commit, that will get rid of this change.